pkg/trace/operation_test.go

```diff
@@ -304,3 +306,25 @@ func TestLogInheritance(t *testing.T) {
 	lm.AssertExpectations(t)
 	lm.AssertNumberOfCalls(t, "Fire", 5)
 }
+
+func TestNewOperationFromID(t *testing.T) {
+	incomingOpID := "101.8"
+	op := NewOperationFromID(context.Background(), &incomingOpID, "TestOperation")
+
+	// Child ID should have three components
+	opIDSlice := strings.Split(op.ID(), ".")
+	require.Equal(t, len(opIDSlice), 3)
```

Comment:
It'd be even better to also verify that `opIDSlice[0]` is `101` and `opIDSlice[1]` is `8`.